CMakeTutorial/MathFunctions/MakeTable.cxx

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#include <math.h>
+#include <iostream>
+
+int main(int argc, char *argv[]) {
+
+    int i;
+    double result;
+
+    if (argc < 2 ) {
+        return 1;
+    }
+
+
+    FILE *fout = fopen(argv[1], "w");
+    if(!fout) {
+        std::cout << "File not found" << std::endl;
```

Comment:
fopen in "w" access mode creates a new file if it does not exist. The error more likely will be caused by the write access restrictions or disk space problem.